packages/clerk-js/scripts/purge-cache.mjs

```diff
@@ -12,10 +12,8 @@ const major = parsedVersion.major;
 try {
   if (!prerelease) {
     await fetch(`https://purge.jsdelivr.net/npm/@clerk/clerk-js@${major}/dist/clerk.browser.js`);
-  } else if (prerelease === 'staging') {
-    await fetch(`https://purge.jsdelivr.net/npm/@clerk/clerk-js@staging/dist/clerk.browser.js`);
   } else {
-    await fetch(`https://purge.jsdelivr.net/npm/@clerk/clerk-js@next/dist/clerk.browser.js`);
+    await fetch(`https://purge.jsdelivr.net/npm/@clerk/clerk-js@canary/dist/clerk.browser.js`);
```

Comment:
❓ Should we use the `clerk-js@${prerelease}/dist/clerk.browser.js` here and in the main (to support the `alpha-v5`  pre-release tag in main).

Reply:
We're going to remove the script completely

Reply:
@dimkl this purge script is actually useless and we will be deleting it soon.